pkg/reporting/preflight.go

```diff
@@ -75,7 +85,19 @@ func SendPreflightInfo(appID string, sequence int, isSkipPreflights bool, isCLI
 				logger.Debugf("failed to get app status: %v", err.Error())
 				return
 			}
-			if s.Sequence == int64(sequence) && s.State == appstatustypes.StateReady {
+
+			currentDeployedSequence, err := store.GetStore().GetCurrentParentSequence(appID, clusterID)
+			if err != nil {
+				logger.Debugf("failed to get downstream parent sequence: %v", err.Error())
+				return
+			}
+
+			if currentDeployedSequence != sequence {
```

Comment:
If I'm calling `SendPreflightInfo` for a sequence, I expect it to report.  Why is this check needed?

Reply:
since the app can take some time to become ready, during that time the user can deploy another version which will make this whole goroutine irrelevant because it's reading a status for a different version. this check will simply not report that case.